src/forms/StripeConnectAccountForm/StripeConnectAccountForm.js

```diff
@@ -35,15 +40,75 @@ const countryCurrency = countryCode => {
 };
 
 const CreateStripeAccountFields = props => {
-  const { disabled, countryLabel, values, intl } = props;
+  const { disabled, countryLabel, showAsRequired, form, values, intl, currentUserId } = props;
+
+  /*
+  We pass some default values to Stripe when creating a new Stripe account in order to reduce couple of steps from Connect Onboarding form.
+  - businessProfileURL: user's profile URL (can be edited through Connect Onboarding)
+  - businessProfileMCC: default MCC code from stripe-config.js (can be edited through Connect Onboarding)
+  - accountToken (https://stripe.com/docs/connect/account-tokens) with following information:
+    * accountType: individual or business
+    * tos_shown_and_accepted: true
+  Only country and bank account token are mandatory values. If you decide to remove the additional default values listed here, remember to update the `createStripeAccount` function in `ducks/stripeConnectAccount.duck.js`.
+  */
+
+  const individualAccountLabel = intl.formatMessage({
+    id: 'StripeConnectAccountForm.individualAccount',
+  });
+
+  const companyAccountLabel = intl.formatMessage({ id: 'StripeConnectAccountForm.companyAccount' });
+
+  const hasBusinessURL = values && values.businessProfileURL;
+  // Use user profile page as business_url on this marketplace
+  // or just fake it if it's dev environment using Stripe test endpoints
+  // because Stripe will not allow passing a localhost URL
+  if (!hasBusinessURL && currentUserId) {
+    const pathToProfilePage = uuid =>
+      createResourceLocatorString('ProfilePage', routeConfiguration(), { id: uuid }, {});
+    const hasCanonicalRootUrl = config && config.canonicalRootURL;
+    const defaultBusinessURL =
+      hasCanonicalRootUrl && !config.canonicalRootURL.includes('localhost')
+        ? `${config.canonicalRootURL}${pathToProfilePage(currentUserId.uuid)}`
```

Comment:
Add check to remove trailing slash if there's one.

Reply:
E.g. https://github.com/sharetribe/ftw-daily/blob/ce9d61adc21df69b5d241d568b01e9dcc05a3b69/src/util/routes.js#L115